draft-btw-add-home.xml

```diff
@@ -1083,27 +1105,19 @@ Legend:
       <t>Hosts serviced by legacy CPEs that can't be upgraded to support the
       options defined in <xref target="RI"></xref> won't be able to learn the
       encrypted DNS server hosted by the ISP, in particular. If the ADN is not
-      discovered using DHCP/RA, such hosts will have to fallback to use the
-      special-use domain name defined in <xref
-      target="I-D.pp-add-resinfo"></xref> to discover the encrypted DNS server
-      and to retrieve the list of supported DoH services using the RESINFO
-      RRtype <xref target="I-D.pp-add-resinfo"></xref>.</t>
-
-      <t>If the Do53 and Encrypted DNS servers are hosted on different public
-      IP addresses, the DNS server certificate can include the IP address of
-      the Do53 server in the subjectAltName extension. After validating the
-      server certificate, the client can retrieve the IP address in the
-      certificate to identify that both Do53 and Encrypted DNS servers are
-      operated by the same entity.</t>
-
-      <t>If the Do53 and Encrypted DNS server are hosted on a private IP
-      address, they should use the same private IP address to prove to the
-      client they are operated by the same entity.</t>
-
-      <t>The DHCP/RA option to discover ADN takes precedence over special-use
-      domain name since the special-use domain name is susceptible to both
-      internal and external attacks whereas DHCP/RA is only vulnerable to
-      internal attacks.</t>
+      discovered using DHCP/RA, such hosts will have to fallback to use DEER 
+      as defined in <xref
+      target="I-D.pauly-add-deer"></xref> to discover the encrypted DNS server
+      and to retrieve the list of supported DoH services using the SVCB
+      RRtype <xref target="I-D.schwartz-svcb-dns"></xref> without verifying the
+	  hostname of discovered templates with the ADN. Other guidance in DEER
+	  relating to resolver verification must be followed in this case.</t>
+
+      <t>The DHCP/RA option to discover DoH templates from DHCP or RA options
```

Comment:
Legacy CPE cannot use DHCP/RA anyway. This paragraph is not appropriate in the Legacy CPE section. Further, DEER will only work for Legacy CPE whose DHCP configuration can be updated to point the clients attached to the home network to ISP Do53 resolver.

Reply:
Nuance: DEER _will_ work for legacy CPE which advertise DNS servers with public IP addresses. Comcast routers handing out 75.75.75.75 is one example. So it is applicable, just in a narrow scope that I think the latest text makes clear.

I think the latest commit's changes to this section will either address this or change what modifications you'll want to see. Either way, please revisit and let me know.